README.adoc

```diff
@@ -33,17 +33,17 @@ Whenever a request is made to the `inventory` service to retrieve the JVM
 system properties of a particular host, the `inventory` service communicates with the `system`
 service on that host to get the system properties. The system properties are then stored and returned.
 
-You will use MicroProfile Metrics to provide endpoints where you can get the metrics
+You will use MicroProfile Metrics have endpoints where you can get the metrics
```

Comment:
The verbs in "You will use MicroProfile Metrics have" are clashing with one another. What about simply deleting the "You will use" part and instead say, "MicroProfile Metrics has endpoints where you can get the metrics..."

Reply:
Yes, that's what I suggested (I thought). It should say, "MicroProfile Metrics have endpoints where you can get the metrics" @Ahmad-Ayyoub pls fix. 
@sishida not sure it'd be "have" or "has" as "Metrics" sounds like plural.

Reply:
@Ahmad-Ayyoub use "has" here.

Reply:
Fixed

Reply:
@evelinec and @Ahmad-Ayyoub: The word "metrics" is plural, so you would want to say, "metrics have." However, isn't "MicroProfile Metrics" a single thing? If so, we'd want "MicroProfile Metrics has."

For example, you would say, "Dunkin Donuts is near my house." and not "Dunkin Donuts are near my house." Even though "Donuts" is plural, you're referring to one shop. Does the same principle apply to "MicroProfile Metrics"?